portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +29,26 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Sean Tisdale!</h1>");
+
+    
+    String json = messages();
+
+   response.setContentType("text/html");
+    response.getWriter().println(json);
+
   }
+
+
+
+    public String messages() {
+    ArrayList<String> message = new ArrayList<String>();
+    message.add("This is my first time using Json");
+    message.add("I wonder if a guy named Jason made Json");
+    message.add("I love pie");
+        return message.get(0);
```

Comment:
Did you mean to only return the first element of the list?

Reply:
Yes. They didn't really say if they wanted the responses to change around as you clicked, but i'll also ask my podmates.

Reply:
I don't think it matters currently.